lib/LANraragi/Controller/Config.pm

```diff
@@ -47,7 +46,7 @@ sub index {
         hqthumbpages    => $self->LRR_CONF->get_hqthumbpages,
         jxlthumbpages   => $self->LRR_CONF->get_jxlthumbpages,
         csshead         => generate_themes_header($self),
-        tempsize        => get_tempsize,
+        tempsize        => 0,
```

Comment:
you can probably discard this value altogether now if it's not used in the templates.